mininterface/textual_interface/textual_adaptor.py

```diff
@@ -42,16 +88,20 @@ def widgetize(tag: Tag) -> Widget | Changeable:
             o = MyCheckbox(tag.name or "", v)
         # Replace with radio buttons
         elif tag._get_choices():
-            o = MyRadioSet(*(RadioButton(label, value=val == tag.val)
-                             for label, val in tag._get_choices().items()))
-        elif isinstance(tag, (SecretTag)):  # NOTE: PathTag, DatetimeTag not implemented
+            o = MyRadioSet(*(RadioButton(label, value=val == tag.val) for label, val in tag._get_choices().items()))
+        elif isinstance(tag, (PathTag, DatetimeTag, SecretTag)):
             match tag:
+                case PathTag():
+                    o = FilePickerInput(
+                        tag, placeholder=tag.name or ""
+                    )
                 case SecretTag():
                     o = SecretInput(tag, placeholder=tag.name or "", type="text")
+                case DatetimeTag():
```

Comment:
let's add a comment

`# NOTE: To be expanded` or something. Because before, a DatetimeTag automatically defaulted to MyInput. You did not seem to change the behaviour.